soundata/core.py

```diff
@@ -6,7 +6,23 @@
 import types
 from typing import Any
 
+from functools import lru_cache
```

Comment:
Could you group imports that are for the figure with a comment? e.g. `# Dataset plots necessary imports`. There are a lot of them...